_posts/0000-01-02-Reuben-Dev.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to Reuben's second slide!"
+---
+Reuben text
```

Comment:
This is excellent 👍 
#1  
![Leben auf den Mars](https://user-images.githubusercontent.com/58612299/77822731-c6a19c80-70f5-11ea-8374-34a727d7bd82.png)